README.md

```diff
@@ -196,6 +196,7 @@ You can pass the following environment variables to LocalStack:
   Kinesis, DynamoDB, Elasticsearch, S3). Set it to `/tmp/localstack/data` to enable persistence
   (`/tmp/localstack` is mounted into the Docker container), leave blank to disable
   persistence (default).
+* `HOST_TMP_FOLDER`: Specifies the main volume if you change it manually (can only be done in the docker-compose file) Default: `/tmp/localstack`) .
```

Comment:
Thanks for adding this @poupougnac ! I'd suggest that we slightly change the description to:
```
* `HOST_TMP_FOLDER`: Path to the temporary folder on the host when running Lambdas in Docker. Defaults to the value of `TMP_FOLDER` if not set.
```

Reply:
The user may think that changing the path will also change the volume path (with localstack --docker). It is necessary to also change the volume path by setting this var.
 I would suggest: 

```
* `HOST_TMP_FOLDER`: Path to the temporary folder on the host when running Lambdas in Docker. Defaults to the value of `TMP_FOLDER` if not set. Don't set it if you don't use a docker-compose file.
```

What do you think ?

Reply:
ok, good point. In this case I'd suggest to write the following:
```
* `HOST_TMP_FOLDER`: Path to the temporary folder on the host when running Lambdas in Docker. Defaults to the value of `TMP_FOLDER` if not set. (Note: changing `HOST_TMP_FOLDER` may require adjusting the volume configuration, either in `docker-compose.yml` or via the `DOCKER_FLAGS` config variable when running `localstack start --docker`.)
```

In fact, I'm wondering if we could add a fix that automatically remaps the volume properly when running `localstack start --docker` in combination with `HOST_TMP_FOLDER`. Thoughts?

Reply:
This would definitely be the best way to go.